regression-tests.recursor-dnssec/test_SNMP.py

```diff
@@ -21,7 +21,7 @@ class TestSNMP(RecursorTest):
     """
 
     def _checkStatsValues(self, results):
-        for i in list(range(1, 93)):
+        for i in list(range(1, 148)):
```

Comment:
looks like this had not been updated in a while. Perhaps it could also check that max+1 does -not- exist? Doesn't have to be in this PR.

Reply:
nice!